typescript/lib/parsers/openai.ts

```diff
@@ -378,7 +379,8 @@ export class OpenAIChatModelParser extends ParameterizedModelParser<Chat.ChatCom
               ? [
                   {
                     output_type: "execute_result",
-                    data: { ...assistantResponse },
+                    data: assistantResponse.content,
+                    metadata: { ...responseWithoutContent },
```

Comment:
no need to spread, it's already a new object
`metadata: responseWithoutContent`